src/Sarif.Viewer.VisualStudio.ResultSources.DeveloperCanvas.Core/Services/DevCanvasResultSourceService.cs

```diff
@@ -144,7 +144,19 @@ public class DevCanvasResultSourceService : IResultSourceService
         /// <inheritdoc/>
         public System.Threading.Tasks.Task InitializeAsync()
         {
-            // TODO: Remove this when merging into main.
+            return IsActiveAsync();
+        }
+
+        /// <inheritdoc/>
+        public Task<Result> IsActiveAsync()
+        {
+            // TODO: remove below to release to general users.
+            string devcanvasKey = (string)Registry.GetValue("HKEY_CURRENT_USER\\Software\\Microsoft\\VisualStudio\\devcanvas", "sampleKey", null);
```

Comment:
Do we want to formalize `sampleKey` a bit more in case we leverage this type of functionality further?  At least name this something more meaningful.

Reply:
Would `DevCanvasInsider` be better?